src/learning_profile.py

```diff
@@ -173,6 +174,6 @@ def __str__(self):
             f"train='{self._train_dataset_name}', "
             f"test='{self._test_dataset_name}', "
             f"al_func={self._al_func.__name__}, "
-            f"ml_func={self._ml_func.__name__}), "
-            f"h_pars={self._hyper_parameters})"
+            f"ml_func={self._ml_func.__name__}, "
+            f"hyper_parameters={self._hyper_parameters})"
```

Comment:
LearningProfile already have support for hyper_parameters from the commit before

Reply:
Fixed!